proposals/0002-ledger-car.md

```diff
@@ -0,0 +1,274 @@
+---
+simd: '0002'
+title: Ledger-CAR file format
+author: Richard Patel (Jump Crypto), Peter Rabbitson (Filecoin), Johnathan Victor (Filecoin)
+category: Interface
+type: Standards Track
+status: Draft
+created: 2022-11-02
+---
+
+## Summary
+
+Ledger-CAR is a streaming file format for Solana ledger data.
+
+Ledger data broadly contains all block information required to validate all state transitions between two slots.
+
+Specifically, this includes two types of data:
+* Proof-of-History parameters
+* Transactions (both user txs and consensus txs)
+
+To support distribution over public networks, Ledger-CAR files are generated deterministically and are stored data via content-addressable IPLD objects.
+
+## Specification
+
+### Solana Entries
+
+The canonical representation of block data is the "entries" format.
+Each entry contains proof-of-history tick parameters and a subslice of transactions recorded in the block.
+
+Entries are serialized as Bincode little-endian encoding.
```

Comment:
This is the canonical serialization format, but not the one actually used in Ledger-CAR. Probably worth removing this line to avoid confusion